lib/content-services/src/lib/common/interfaces/search-configuration.interface.ts

```diff
@@ -27,6 +26,5 @@ export interface SearchConfigurationInterface {
      * @param skipCount The offset of the start of the page within the results list
      * @returns Query body defined by the parameters
      */
-    generateQueryBody(searchTerm: string, maxResults: number, skipCount: number): QueryBody;
-
+    generateQueryBody(searchTerm: string, maxResults: number, skipCount: number): SearchRequest;
```

Comment:
@DenysVuika there is search-configuration.interface.md file which still has QueryBody instead SearchRequest. Could you please correct that documentation file too to have it synchronized?

Reply:
I can't change the public api yet as it's going to be a breaking change, while changing the return type seems to be relatively ok